### ISSUE TITLE
Fel Reavers, No Thanks! (10855) (BB #933)

### ISSUE THREAD

**Issue by OregonAsari** [OregonCore/OregonCore#933]
This issue was migrated from bitbucket.
**Original Reporter:** bewtihozzy
**Original Date:** 29.05.2015 20:50:07 GMT+0000
**Original Priority:** major
**Original Type:** bug
**Original State:** resolved
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/933

<hr>

Fel Reavers, No Thanks! (10855)
Obtain 5 Condensed Nether Gas from Gan'arg Mekgineers at Forge Base: Oblivion, northwest of the Stormspire, and load them into a nearby Inactive Fel Reaver

bug: When i load the gas into a nearby Inactive Fel Reaver it shows Quest complete but in the quest log its not completed.

More info here: [video](https://youtu.be/lnDOEW1NFoo)

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Selphius
**Original Date:** 22.11.2015 18:43:25 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/933/#comment-23498718

<hr>

Aaaannndddd another fix:

```
#!c++

/*######
## QUEST_FEL_REAVER'S NO THANKS
######*/

#define QUEST_NETHER_GAS_IN_FEL_FIRE_ENGINE 10850
#define QUEST_FEL_REAVER_NO_THANKS 10855
#define INACTIVE_TEXT "Inactive Fel Reaver begins to sputter as it's engine malfunctions."

//Keep in mind that is is only a temporary solution, it will complete the quest, but not the way it should. Also there seems to be some sort of visual problem -> quest does complete but in quest log it isn't shown that it is completed.

bool ChooseReward_npc_inactive_fel_reaver(Player* pPlayer, Creature* pCreature, const Quest* pQuest, uint32 /*item*/)
{
    if (pQuest->GetQuestId() == QUEST_NETHER_GAS_IN_FEL_FIRE_ENGINE)
    {   
        pCreature->MonsterTextEmote(INACTIVE_TEXT, 0, false);
        pPlayer->CompleteQuest(QUEST_FEL_REAVER_NO_THANKS);
    }
    return true;
}

newscript = new Script;
    newscript->Name = "npc_inactive_fel_reaver";
    newscript->pChooseReward = &ChooseReward_npc_inactive_fel_reaver;
    newscript->RegisterSelf();
```

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** AlphafoxBHN
**Original Date:** 22.11.2015 19:31:01 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/933/#comment-23499233

<hr>

sorry big brain lag ... how i use this ?

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 25.12.2015 02:23:35 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/933/#comment-24313770

<hr>

[DB/SAI] Script support for quest Fel Reavers, No Thanks!
         Fixes #933

→ <<cset 1cacfd0d8720>>
